README.md

```diff
@@ -1,4 +1,5 @@
-## :information_source: If you are using OCP 4.6 please follow this [README](https://github.com/openshift/kata-operator/blob/release-4.6/README.md)
+## :information_source: If you are using OCP 4.7 please follow this [README](https://github.com/openshift/sandboxed-containers-operator/blob/release-4.7/README.md)
+## :information_source: If you are using OCP 4.6 please follow this [README](https://github.com/openshift/sandboxed-containers-operator/blob/release-4.6/README.md)
 
 # Kata Operator
```

Comment:
Should reference to "Kata Operator" in the above section and following instructions be changed as well ?

Reply:
At first I only wanted to change URLs, but you're right why not change the name of the operator in the text as well. I changed it from "Kata Operator" to "OpenShift sandboxed containers operator" now.